molecule/default/verify.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify role
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Ensure opendkim service is enabled and running
+      service:
```

Comment:
A `service` task isn't something we can use to prove that the role did its job. If the service isn't already running, this simply starts it. 

If a test changes the state of something we're trying to test, then the results won't be accurate.

Yes, this task would fail if the service wasn't installed, but so would the [playbook](https://github.com/AcroMedia/ansible-role-dkim/blob/master/tasks/opendkim.yml#L125), so this task would never be encountered.

The best way to assert that a service is running, is to try and use it, and then gather evidence that the service did what it was supposed to. In this case that might get complicated, because it depends on postfix being installed too. 

A more trivial test would be to simply check that its [tcp port](https://github.com/AcroMedia/ansible-role-dkim/blob/master/defaults/main.yml#L9) is listening, using ss or netstat,

Reply:
Modified.